Tests/W3CTestSuite/svg/__pull_request-963-01.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="210mm"
```

Comment:
I tested this file in master branch. I opened this file using svgdocument.open method. And argumentexception occured when svgdocument.Bounds was called. so I fixed this issue in this pull request.
Sorry, i'm not good at english.

Reply:
> when svgdocument.Bounds was called.

This means you called it, right ?
Please change these(width and height) to percentage as it will not be called in automated test (it will not be a test).